internal/http/services/owncloud/ocdav/ocdav.go

```diff
@@ -175,9 +175,32 @@ func (s *svc) Handler() http.Handler {
 			// oc uses /dav/files/$user -> /$home/$user/...
 			// for oc we need to prepend the path to user homes
 			// or we take the path starting at /dav and allow rewriting it?
+
```

Comment:
This PR finally addresses the above comments. The /webdav and dav/files/<userid> endpoints need to show the same tree for a user. /dav/files/<userid> can also use the userid for other users, which is currently forbidden by oc10 but for ocis access is in theory possible using the eos driver because access is based on acls.